autotest/common_regression.py

```diff
@@ -160,15 +161,15 @@ def setup_comparison(namefile, dst, remove_existing=True):
                 print("Could not make " + dst)
         # clean directory
         else:
-            print(f"cleaning...{dst}")
+            print("Cleaning ", dst)
```

Comment:
The change is fine but I am wondering if there is a reason to move away from f-strings